src/Symfony/Component/Serializer/Normalizer/MimeMessageNormalizer.php

```diff
@@ -42,6 +42,17 @@ public function __construct(PropertyNormalizer $normalizer)
         $this->headersProperty = new \ReflectionProperty(Headers::class, 'headers');
     }
 
+    public function getSupportedTypes(): ?array
```

Comment:
We could make all concrete implementations declare only `array` as return type (not nullable)

Reply:
done